### ISSUE TITLE
Pickles UI DHTML Site differs from Command Line DHTML Site

### ISSUE THREAD

**Issue by jmb-mage** [picklesdoc/pickles#524]
tl;dr - I am hoping I am just missing a param in my command line.

Here my PicklesUI settings:

![picklesui](https://user-images.githubusercontent.com/4942457/39251804-ad0318c6-4893-11e8-8ad5-383d552efedc.PNG)

It's pointing at the same TestResults file and *I think* has the same settings as the command line I run:
`C:\Program Files (x86)\Pickles\pickles.exe --feature-directory=C:\dev\AcceptanceTests\AcceptanceTests\Features --output-directory=C:\dev\TestOutput\AcceptanceTests\Pickles --DocumentationFormat dhtml --EnableComments true --link-results-file C:\dev\AcceptanceTests\AcceptanceTests\CI\TestResult.xml --test-results-format nunit --exp yes --system-under-test-name  Acceptance Tests --system-under-test-version 1.0.0`

Pickles v.2.18.1.0

Generating documentation based on the following parameters
----------------------------------------------------------
Feature Directory              : C:\dev\AcceptanceTests\AcceptanceTests\Features
Output Directory               : C:\dev\TestOutput\AcceptanceTests\Pickles
Project Name                   : Acceptance Tests
Project Version                : 1.0.0
Language                       : en
Incorporate Test Results?      : Yes
Include Experimental Features? : Yes
Exclude Tag                    :
Test Result Format             : NUnit
Test Result File               : C:\dev\AcceptanceTests\AcceptanceTests\CI\TestResult.xml
Writing HTML to C:\dev\TestOutput\AcceptanceTests\Pickles
Pickles completed successfully

## Differences in output ## 

PicklesUI Folder:
![picklesuifolder](https://user-images.githubusercontent.com/4942457/39252252-cd3d8d46-4894-11e8-91c8-3cccc47d97e7.PNG)

PicklesUI Site:
![picklesuisite](https://user-images.githubusercontent.com/4942457/39252316-ecd2c612-4894-11e8-8e91-9b9e2ebe12ed.PNG)

Pickles Command Line Folder:
![picklesclifolder](https://user-images.githubusercontent.com/4942457/39252363-053c24d2-4895-11e8-93fd-1e35e81a29cd.PNG)

Pickles Command Line Site:
![picklesclisite](https://user-images.githubusercontent.com/4942457/39252416-1f415078-4895-11e8-833e-30246a8421fb.PNG)

**Comment by dirkrombauts** [comment]
Hi,

In the UI you are generating two output formats: Html and DHtml. Those two will overwrite one another - in this case the Dhtml version won.

In the command line version, you are specifying `dhtml`. I think the enumeration value might be case sensitive, so please trying specifying `DHtml`. `dhtml` isn't recognized, so pickles falls back to the default output, which it `Html`.

BTW: If you do want to generate multiple output formats, you have two options: call the command line version multiple times, or use the UI version of pickles with multiple output formats selected _and_ `create folder for each output format` activated.

Does that answer your question?

**Comment by dirkrombauts** [comment]
BTW: excellent issue report. Lots of command lines, lots of screenshots. That's the way I like it :-)

**Comment by jmb-mage** [comment]
As soon as you said "defaults" and "case sensitive" I figured it out.

My CLI had this `--DocumentationFormat dhtml` instead of this `--documentation-format=dhtml`

I fixed that and it worked!  Thanks.

ps - Thanks for the compliment!  I work in QA so I want to know all the deets.  And thanks again for PicklesDoc.
